modules/ROOT/pages/to-set-permissions.adoc

```diff
@@ -4,35 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Exchange provides multiple ways to set up access management to meet governance requirements and at the same time drive collaboration across teams. Anypoint Platform comes with predefined role groups that can be set in the Access Management > Roles settings to provide users with different access levels.
+Exchange provides multiple ways to set up access management to provide collaboration across teams and meet governance requirements. Anypoint Platform comes with predefined permissions and roles that can be set in the Access Management section of Anypoint Platform to provide users with different access levels.
 
-== Exchange Role Groups
+== Exchange Permissions
 
-These role groups pertain to every Business Group level:
+To control access to assets in a business group, add users or teams to Exchange permissions role groups.
```

Comment:
"permissions roles groups" sounds overloaded: permissions groups? role groups? roles?

Reply:
Changed to "role groups for Exchange permissions."